src/Microsoft.TestPlatform.Client/Execution/TestRunRequest.cs

```diff
@@ -190,12 +190,12 @@ internal void OnTestSessionTimeout(object obj)
         {
             if (EqtTrace.IsVerboseEnabled)
             {
-                EqtTrace.Verbose(String.Format("TestRunRequest.OnTestSessionTimeout: calling cancellation as test run exceeded testSessionTimeout {0} milliseconds", testSessionTimeout));
+                EqtTrace.Verbose(string.Format("TestRunRequest.OnTestSessionTimeout: calling cancellation as test run exceeded testSessionTimeout {0} milliseconds", testSessionTimeout));
```

Comment:
We should put that string into `ClientResources` to allow l10n.

Reply:
But this is EqTrace message, we do not add them with localization. It used for logging and available only in English.